### ISSUE TITLE
lense correction, camera/lens not found

### ISSUE THREAD

**Issue by ptilopteri** [darktable-org/darktable#11050]
openSUSE Tumbleweed 20220201
NVIDIA GF106 [GeForce GTS 450], 390.147
darktable 3.9.0+202~g6ef3f6b05
bit depth is 64 bit
normal build
SSE2 optimized codepath enabled
OpenMP support enabled
OpenCL support enabled
Lua support enabled, API version 6.1.0
Colord support enabled
gPhoto2 support enabled
GraphicsMagick support enabled
ImageMagick support disabled
OpenEXR support enabled
OpenCL on
i7 12-core 970
ram 36GB
5.16.2-1-default

from exif:
    Camera Model Name               : NIKON D850
    Creator Tool                    : NIKON D850 Ver.1.20
    Lens Focus Function Buttons     : AF Lock Only
    Lens Data Version               : 0204
    Lens ID Number                  : 183
    Lens Type                       : G VR
    Lens                            : 80-400mm f/4.5-5.6
    Lens F Stops                    : 5.67
    Lens ID                         : AF-S Nikkor 80-400mm f/4.5-5.6G ED VR
    Lens Spec                       : 80-400mm f/4.5-5.6 G VR
    
dt reports:
    [lens correction] camera/lens not found
    [lens correction] camera/lens not found
    [lens correction] camera/lens not found

this camera/lense combination worked previously and both camera and lens are in:
    /usr/share/lensfun/version_1/slr-nikon.xml
    /var/lib/lensfun-updates/version_2/slr-nikon.xml

the camera/lens combination should be found and correction applied as previously.

tks

**Comment by ptilopteri** [comment]
sample file with failing message:
  http://wahoo.no-ip.org/~paka/220202_105742_aaa_7801.nef

have 28 more that exhibit the same failure

**Comment by kmilos** [comment]
Identified by exiv2 as

```
Exif.NikonLd3.LensIDNumber                    Nikon AF-S Nikkor 80-400mm f/4.5-5.6G ED VR
```

I don't see this lens in the most up-to-date `slr-nikon.xml`

**Comment by kmilos** [comment]
Indeed, this lens has not been added yet, see https://github.com/lensfun/lensfun/issues/1501

Not sure how it worked for your previously...

**Comment by PeterWem** [comment]
Manually copy the data for the lens into your own slr-nikon.xml file.

**Comment by ptilopteri** [comment]
where do I put "my own" slr-nikon.xml file.  Sorry, I did know but
cannot remember, old age.

**Comment by ptilopteri** [comment]
I created an xml file by copying your https://github.com/lensfun/lensfun/issues/1501 
and saving to ~/.local/share/lensfun as described on pixls.us,
Create lens calibration data for lensfun
but still get camera/lens not found

changed it to ~/.local/share/lensfun.xml
and still get camera/lens not found

changed it to ~/local/share/lensfun/lensfun.xml
and success

tks much

ps: I commented on pixls.us the corrected location

**Comment by PeterWem** [comment]
Your slr-nikon.xml file should be in 

/usr/share/lensfun/version_1 
or 
/var/lib/lensfun-updates/version_1 
or 
/.local/share/lensfun

Take a backup of slr-nikon.xml. Then open slr-nikon.xml and paste the data.

**Comment by ptilopteri** [comment]
I copied your data to a file, lensfun.xml in ~/.local/share/lensfun/

the file, lensfun.xml contains *only* the data you provided

and works fine, tks
